org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/TelemetryManager.java

```diff
@@ -106,9 +114,36 @@ public void onBuildFinished(long buildFinishedTime) {
 						sourceLevelMax = Float.parseFloat(sourceLevel);
 					}
 				}
+
+				try {
+					IMarker[] projectMarkers = project.findMarkers(null /* all markers */, true /* subtypes */, IResource.DEPTH_ZERO);
+					projectErrors += Stream.of(projectMarkers).filter(m -> m.getAttribute(IMarker.SEVERITY, 0) == IMarker.SEVERITY_ERROR).count();
+					IMarker[] allmarkers = project.findMarkers(null /* all markers */, true /* subtypes */, IResource.DEPTH_INFINITE);
```

Comment:
Is it worth using `ResourceUtils.getErrorMarkers(project)` here ? At least for the infinite depth case.

Reply:
Using the helper method does not seem to save code. I prefer to keep the current code as it is.